<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtEEG-Map</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/icon.svg</normaloff>:/icon/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>553</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="title">
          <string>Domain</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioTime">
            <property name="text">
             <string>Tegangan</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioFrequencyMultiple">
            <property name="text">
             <string>Frekuensi Ganda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioFrequencySingle">
            <property name="text">
             <string>Frekuensi Tunggal</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Elektroda</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="checkboxTP9">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>TP9</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxAF7">
            <property name="text">
             <string>AF7</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxAF8">
            <property name="text">
             <string>AF8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxTP10">
            <property name="text">
             <string>TP10</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Pita Frekuensi</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="checkboxGamma">
            <property name="text">
             <string>Gamma</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxBeta">
            <property name="text">
             <string>Beta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxAlpha">
            <property name="text">
             <string>Alpha</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxTheta">
            <property name="text">
             <string>Theta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkboxDelta">
            <property name="text">
             <string>Delta</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>88</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="GraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;Dokumen</string>
    </property>
    <widget class="QMenu" name="menuConnect_do_device">
     <property name="title">
      <string>&amp;Perangkat</string>
     </property>
     <addaction name="actionStream"/>
     <addaction name="actionDisconnect"/>
     <addaction name="separator"/>
     <addaction name="actionRecord"/>
     <addaction name="actionStop"/>
     <addaction name="actionView"/>
    </widget>
    <addaction name="menuConnect_do_device"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Bantuan</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuSpikes">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionSpike_sorting"/>
    <addaction name="actionSpike_detecting"/>
    <addaction name="actionFeature_extraction"/>
    <addaction name="actionClustering"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Alat</string>
    </property>
    <addaction name="actionStimuli"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuSpikes"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllToolBarAreas</set>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionStream"/>
   <addaction name="actionDisconnect"/>
   <addaction name="separator"/>
   <addaction name="actionRecord"/>
   <addaction name="actionStop"/>
   <addaction name="actionView"/>
   <addaction name="separator"/>
   <addaction name="actionStimuli"/>
   <addaction name="separator"/>
   <addaction name="actionSpike_sorting"/>
   <addaction name="actionSpike_detecting"/>
   <addaction name="actionFeature_extraction"/>
   <addaction name="actionClustering"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normalon>:/icon2/open2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Buka Dokumen</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Keluar</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>Tentang</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Bantuan</string>
   </property>
  </action>
  <action name="actionStream">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/connect2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Hubungkan dan Si&amp;arkan</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/disconnect2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Putuskan</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/record2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Rekam</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/stop2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Berhenti</string>
   </property>
  </action>
  <action name="actionView">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/view2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Lihat</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Pengaturan</string>
   </property>
  </action>
  <action name="actionStimuli">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/stimuli2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Stimuli</string>
   </property>
  </action>
  <action name="actionSpike_sorting">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/sorting2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Pengurutan Lonjakan Sinyal</string>
   </property>
  </action>
  <action name="actionSpike_detecting">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/detection2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Deteksi Lonjakan Sinyal</string>
   </property>
  </action>
  <action name="actionFeature_extraction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/feature_selection2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Ekstraksi Fitur</string>
   </property>
  </action>
  <action name="actionClustering">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icon2/clustering2.svg</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Pengelompokan</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph/widgets/GraphicsView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
